pylxd/connection.py

```diff
@@ -22,6 +22,22 @@
 from pylxd import utils
 from six.moves import http_client
 
+if hasattr(ssl, 'SSLContext'):
```

Comment:
I think it'd be better to try/except here, rather than the if. It's semantic, so not really a big deal, but a more "pythonic" solution.